proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -135,7 +169,10 @@ service QueryService {
   rpc ReadUtxos(ReadUtxosRequest) returns (ReadUtxosResponse); // Read specific UTxOs by reference.
   rpc SearchUtxos(SearchUtxosRequest) returns (SearchUtxosResponse); // Search for UTxO based on a pattern.
   rpc ReadData(ReadDataRequest) returns (ReadDataResponse); // Read specific datum by hash
-  rpc ReadTx(GetTxRequest) returns (GetTxResponse); // Get Txs by by chain-specific criteria.
+  rpc ReadTx(ReadTxRequest) returns (ReadTxResponse); // Get Txs by chain-specific criteria.
+  rpc ReadEraSummary(ReadEraSummaryRequest) returns (ReadEraSummaryResponse); // Get summary of all eras
+  rpc ReadEraConfig(ReadEraConfigRequest) returns (ReadEraConfigResponse); // Get configuration for an era
+  rpc ReadChainId(ReadChainIdRequest) returns (ReadChainIdResponse); // Get the chain ID for this network
```

Comment:
@SebastienGllmt do we need to split this three concerns into different methods?

I feel that many of these calls would probably go together in most scenarios and having them separated would make the interface too chatty.

Maybe having something like a single `ReadChain` method and rely on a nested structure that we can filter using field_masks.

Something like:

- chain
  - genesis
  - caip2
  - eras[]
    - name
    - start
    - end
    - config
 
What are your thoughts?

Reply:
I'm totally fine with that as well

Reply:
@scarmuega how do you want to do the bit mask for `config` given it's nested in the `eras` array?

Reply:
Let me know if #151 is what you were thinking of